src/Mod/Sketcher/Gui/DrawSketchHandlerArc.h

```diff
@@ -84,6 +85,45 @@ class DrawSketchHandlerArc: public DrawSketchHandlerArcBase
     ~DrawSketchHandlerArc() override = default;
 
 private:
+    std::list<Gui::InputHint> getToolHints() const override
+    {
+        using Gui::InputHint;
+        using enum Gui::InputHint::UserInput;
+
+        if (constructionMethod() == ConstructionMethod::Center) {
+            switch (state()) {
+                case SelectMode::SeekFirst:
+                    return {InputHint {QObject::tr("%1 pick arc center"), {MouseLeft}},
+                            InputHint {QObject::tr("%1 switch mode"), {KeyM}}};

```

Comment:
This hint is very often repeated. I wonder if there is a way to not repeat it's definition so much? For now you can just define `const InputHint switchModeHint {QObject::tr("%1 switch mode"), {KeyM}}` on the root level of function and use it in the lists.